src/axom/slic/docs/sphinx/sections/architecture.rst

```diff
@@ -0,0 +1,453 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _sections/architecture:
+
+Component Architecture
+-----------------------
+
+Slic provides a simple and easy to use logging interface for applications.
+
+.. _figs/slic_architecture:
+
+.. figure:: ../figures/architecture.png
```

Comment:
I can't put a comment on the image file, so I'll put it here.

The yellow "level" is oddly offset from the grey "msg."  Shouldn't they have the same baseline?

Reply:
Done.